.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Python 3 on Linux
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.6, 3.7]
```

Comment:
are we dropping python 2 at this point?

Reply:
Why should we include it here? It is already in Travis and since we're going to be dropping Python 2.7 at the end of the year we can just start fresh on a new CI platform and let Travis deal with 2.7 until Python 3 freedom day.